tests/kernel/smp/testcase.yaml

```diff
@@ -14,3 +14,11 @@ tests:
     filter: (CONFIG_MP_MAX_NUM_CPUS > 1) and CONFIG_MINIMAL_LIBC_SUPPORTED
     extra_configs:
       - CONFIG_MINIMAL_LIBC=y
+  kernel.multiprocessing.smp.affinity:
+    tags:
+      - kernel
+      - smp
+    ignore_faults: true
+    filter: (CONFIG_MP_MAX_NUM_CPUS > 1)
+    extra_configs:
+      - CONFIG_SCHED_CPU_MASK=y
```

Comment:
The `CONFIG_SCHED_CPU_MASK` depends on `CONFIG_SCHED_DUMB`. As I can see some boards selects other SCHED options - so I'm not sure if it'll work properly. Do we need to add proper filtering here?

Reply:
I remember this will be resolved by twister.